app/scss/_campussearch.scss

```diff
@@ -31,7 +31,7 @@ $campussearch-max-width: 29em;
   }
 
   @include bp(screen2) {
-    font-size: 2.6vw;
+    font-size: 2.0vw;
```

Comment:
Justin, you need to feed this back to Joel as well; otherwise it'll get overridden next time we integrate.